nav2_msgs/msg/PolygonVO.msg

```diff
@@ -0,0 +1,5 @@
+std_msgs/Header header
+geometry_msgs/Point32[] points
+bool closed
+int8 value
+unique_identifier_msgs/UUID uuid
```

Comment:
Move uuid fields right after `header` in both messages. Just a formatting thing

Reply:
Done for [CircleObject.msg](https://github.com/ros-planning/navigation2/pull/3930/commits/d6a831d90d1440436538a61c771f017bddb59422#diff-455a4f390eb3e15399b91e734fd767cd265c8f21b9736f4cc3694468b46f1acd) and [PolygonObject.msg](https://github.com/ros-planning/navigation2/pull/3930/commits/d6a831d90d1440436538a61c771f017bddb59422#diff-69588642e05a866e11ca8c89c7c583cb4c4eef4ca26f25f71b4b1fa5180ffbdc)